Marlin/language_ru.h

```diff
@@ -171,5 +261,110 @@
+#define MSG_DELTA_HEIGHT_CALIBRATE          _UxGT("Задать высоту Delta")
+#define MSG_INFO_MENU                       _UxGT("О принтере")
+#define MSG_INFO_PRINTER_MENU               _UxGT("Информация о принтере")
+#define MSG_3POINT_LEVELING                 _UxGT("Выравнивание по 3 точкам")
+#define MSG_LINEAR_LEVELING                 _UxGT("Линейное выравнивание")
+#define MSG_BILINEAR_LEVELING               _UxGT("Квадратичное выравнивание")
+#define MSG_UBL_LEVELING                    _UxGT("Универсальное выравниваие")
+#define MSG_MESH_LEVELING                   _UxGT("Выравнивание по сетке")
+#define MSG_INFO_STATS_MENU                 _UxGT("Статистика принтера")
+#define MSG_INFO_BOARD_MENU                 _UxGT("Информация о контроллере")
+#define MSG_INFO_THERMISTOR_MENU            _UxGT("Термисторы")
+#define MSG_INFO_EXTRUDERS                  _UxGT("Экструдеры")
+#define MSG_INFO_BAUDRATE                   _UxGT("Бод")
+#define MSG_INFO_PROTOCOL                   _UxGT("Протокол")
+#define MSG_CASE_LIGHT                      _UxGT("Корпусное освещение")
+#define MSG_CASE_LIGHT_BRIGHTNESS           _UxGT("Яркость освещения")
+#if LCD_WIDTH >= 20
+  #define MSG_INFO_PRINT_COUNT              _UxGT("Количество отпечатков")
+  #define MSG_INFO_COMPLETED_PRINTS         _UxGT("Закончено")
+  #define MSG_INFO_PRINT_TIME               _UxGT("Полное время печати")
+  #define MSG_INFO_PRINT_LONGEST            _UxGT("Максимальное время печати")
+  #define MSG_INFO_PRINT_FILAMENT           _UxGT("Длинна филамента")
+#else
+  #define MSG_INFO_PRINT_COUNT              _UxGT("Отпечатков")
+  #define MSG_INFO_COMPLETED_PRINTS         _UxGT("Закончено")
+  #define MSG_INFO_PRINT_TIME               _UxGT("Всего")
+  #define MSG_INFO_PRINT_LONGEST            _UxGT("Наибольшее")
+  #define MSG_INFO_PRINT_FILAMENT           _UxGT("Выдавлено")
+#endif
+#define MSG_INFO_MIN_TEMP                   _UxGT("Минимальная температура")
+#define MSG_INFO_MAX_TEMP                   _UxGT("Максимальная температура")
+#define MSG_INFO_PSU                        _UxGT("Блок питания")
+#define MSG_DRIVE_STRENGTH                  _UxGT("Сила привода")
+#define MSG_DAC_PERCENT                     _UxGT("Привод %")
+#define MSG_DAC_EEPROM_WRITE                _UxGT("Записи DAC EEPROM")
+#define MSG_FILAMENT_CHANGE_HEADER          _UxGT("ПЕЧАТЬ ОСТАНОВЛЕНА")
+#define MSG_FILAMENT_CHANGE_OPTION_HEADER   _UxGT("ВАРИАНТЫ ПРОДОЛЖЕНИЯ:")
+#define MSG_FILAMENT_CHANGE_OPTION_EXTRUDE  _UxGT("Выдавить ещё")
+#define MSG_FILAMENT_CHANGE_OPTION_RESUME   _UxGT("Возобновить печать")
+#define MSG_FILAMENT_CHANGE_MINTEMP         _UxGT("Минимальная температура ")
+#define MSG_FILAMENT_CHANGE_NOZZLE          _UxGT("  Сопла: ")
+//
+// Filament Change screens show up to 3 lines on a 4-line display
+//                        ...or up to 2 lines on a 3-line display
+//
+#if LCD_HEIGHT >= 4
+  #ifndef MSG_FILAMENT_CHANGE_INIT_1
```

Comment:
Don't include the `#ifndef`/`#endif` lines in this section. They must only be included in `language_en.h`. Instead, just define them. For example…
```cpp
#if LCD_HEIGHT >= 4
  #define MSG_FILAMENT_CHANGE_INIT_1      _UxGT("Ожидайте")
  #define MSG_FILAMENT_CHANGE_INIT_2      _UxGT("начала смены")
  #define MSG_FILAMENT_CHANGE_INIT_3      _UxGT("филамента")
  #define MSG_FILAMENT_CHANGE_UNLOAD_1    _UxGT("Ожидайте")
  #define MSG_FILAMENT_CHANGE_UNLOAD_2    _UxGT("выгрузки филамента")
  #define MSG_FILAMENT_CHANGE_INSERT_1    _UxGT("Вставьте филамент")
  #define MSG_FILAMENT_CHANGE_INSERT_2    _UxGT("и нажмите кнопку")
  #define MSG_FILAMENT_CHANGE_INSERT_3    _UxGT("для продолжения...")
  #define MSG_FILAMENT_CHANGE_HEAT_1      _UxGT("Нажмите кнопку для")
  #define MSG_FILAMENT_CHANGE_HEAT_2      _UxGT("нагрева сопла...")
  #define MSG_FILAMENT_CHANGE_HEATING_1   _UxGT("Нагрев сопла")
  #define MSG_FILAMENT_CHANGE_HEATING_2   _UxGT("Ждите...")
  #define MSG_FILAMENT_CHANGE_LOAD_1      _UxGT("Ожидайте")
  #define MSG_FILAMENT_CHANGE_LOAD_2      _UxGT("загрузки филамента")
  #define MSG_FILAMENT_CHANGE_RESUME_1    _UxGT("Ожидайте")
  #define MSG_FILAMENT_CHANGE_RESUME_2    _UxGT("возобновления")
#else // LCD_HEIGHT < 4
  #define MSG_FILAMENT_CHANGE_INIT_1      _UxGT("Ожидайте...")
  #define MSG_FILAMENT_CHANGE_UNLOAD_1    _UxGT("Выгрузка...")
  #define MSG_FILAMENT_CHANGE_INSERT_1    _UxGT("Вставь и нажми")
  #define MSG_FILAMENT_CHANGE_HEATING_1   _UxGT("Нагрев...")
  #define MSG_FILAMENT_CHANGE_LOAD_1      _UxGT("Загрузка...")
  #define MSG_FILAMENT_CHANGE_EXTRUDE_1   _UxGT("Выдавливание...")
  #define MSG_FILAMENT_CHANGE_RESUME_1    _UxGT("Возобновление...")
#endif // LCD_HEIGHT < 4
```